src/app/modules/shared_models/message_item.nim

```diff
@@ -50,6 +50,8 @@ type
     quotedMessageParsedText: string
     quotedMessageContentType: int
     quotedMessageDeleted: bool
+    quotedMessageAuthorDisplayName: string
+    quotedMessageAuthorAvatar: string
     # This is only used to update the author's details when author's details change
     quotedMessageFromIterator: int
```

Comment:
~~Maybe we no longer need the iterator if we set it directly in the message. I don't know why I didn't just do that myself TBH. I knew relying on a JSON get was bad anyway~~

Ok I see that those apply only for Discord messages for now. IMO, we should improve that. I'll open an issue to always use your new properties